physionet-django/console/templates/console/event_management_manage_dataset.html

```diff
@@ -4,6 +4,9 @@
     <ul class="list-group list-group-flush">
       <li class="list-group-item">
         <h5>Add a dataset to the event</h5>
+          <p>By adding a dataset, all participants (approved by {{ event.host.username }} or their cohosts) will be allowed to access this
```

Comment:
This was a comment by Benjamin, requesting it to be made clear so that event host who adds the data acknowledges the fact that he is allowing the participants to access the data directly without any trainings for the duration of the event.